crates/cubecl-linalg/src/convolution/reader/im2col.rs

```diff
@@ -149,3 +184,9 @@ impl<E: Numeric> Im2colReader<E> {
         self.tensor.read(position)
     }
 }
+
+#[allow(unused_variables)]
+#[cube]
+fn unwrap(v: u32) -> comptime_type!(u32) {
+    intrinsic!(|_| v.constant().expect("Must be constant").as_u32())
+}
```

Comment:
We could have a function for index (u32) which is `u32.as_const_unchecked() -> u32`

Reply:
I think so too, but I'd do it in a different PR probably. Alternatively, we might want to just take an expanded u32 and unwrap like we do with some other things rn.